shared/src/main/java/uk/gov/di/authentication/shared/services/ConfigurationService.java

```diff
@@ -514,4 +514,26 @@ public long getAccountInterventionServiceCallTimeout() {
     public String getAccountInterventionsErrorMetricName() {
         return System.getenv().getOrDefault("ACCOUNT_INTERVENTIONS_ERROR_METRIC_NAME", "");
     }
+
+    public String getIPVAudience() {
```

Comment:
Could you leave a comment on the ticket that uses this to pass this variable please?

Reply:
[Done](https://govukverify.atlassian.net/browse/ATO-660?focusedCommentId=152305)